mate-base/mate-menus/mate-menus-1.26.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GNOME2_LA_PUNT="yes"
+
+inherit mate
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+fi
+
+DESCRIPTION="MATE menu system, implementing the F.D.O cross-desktop spec"
+LICENSE="GPL-2+ LGPL-2+ LGPL-2.1+"
+SLOT="0"
+
+IUSE="debug +introspection nls"
+
+COMMON_DEPEND=">=dev-libs/glib-2.50:2
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+"
+
+RDEPEND="${COMMON_DEPEND}"
+
+DEPEND="${COMMON_DEPEND}
+	>=sys-devel/gettext-0.19.8
+	virtual/pkgconfig
+"
+
+src_configure() {
+	# Do NOT compile with --disable-debug/--enable-debug=no as it disables API
```

Comment:
Now I'm wondering if all MATE packages should not be doing --disable-debug?

Reply:
?

Reply:
I meant: can this be a use flag?
But it's already a use flag, just not enabled by default.
As for your question, I don't know. Most of the ebuild where written by NP-Hardass and I am only humble servant.
I don't understand everything he did. I slowly get it, but the intervals between my updates are long ...

Reply:
What I'm saying is: this package has some special logic for debug. We should, at some point, look into whether other MATE packages should copy this logic.

No need to worry about it right now if you're not sure. :)